rescript/plugin_setup.py

```diff
@@ -289,7 +289,13 @@
                 'ancestor among all taxa sharing a sequence. "majority" will '
                 'find the most common taxonomic label associated with that '
                 'sequence; note that in the event of a tie, "majority" will '
-                'pick the winner arbitrarily.',
+                'pick the winner arbitrarily. "super" finds the LCA '
+                'consensus while giving preference to majority labels and '
```

Comment:
if any of this text is redundant with the text for `merge_taxa`, I recommend just defining the text string as a variable and appending in both descriptions rather than writing redundant lines of text.